src/formio/components/FileField.stories.mdx

```diff
@@ -49,6 +61,9 @@ export const ERRORS = ['File is too big', 'File is of the wrong type'];
 **Note** - file uploads on the Docs page will always error due to the way
 the network mocking library works. Use the canvas instead.
 
+By selecting a `filePattern` and uploading a file of different type, it is possible to see the
+error raised when a user uploads a file of wrong type.
```

Comment:
Can you please implement a `play` function/interaction test that asserts the necessary error messages and formats are displayed?